test/aspnettests.js

```diff
@@ -115,10 +115,10 @@ describe('ASP.NET project file creation when kestrel command exists', function (
     before(function (done) {
         helpers.run(path.join( __dirname, '../generators/app'))
         .inTmpDir(function(dir) {
-            createTestProjectJson(dir, 'EXISTING_KESTREL_COMMAND'); })
+            createTestProjectJson(dir, 'EXISTING_WEB_COMMAND'); })
         .withLocalConfig(function() {
             return { "appInsightsOptIn": false, "runningTests": true }; })
-        .withPrompts({ projectType: 'aspnet', aspNetVersion: '1.0.0-beta8' })
+        .withPrompts({ projectType: 'aspnet', aspNetVersion: '1.0.0-rc1-final' })
```

Comment:
We have used the same solution on `generator-aspnet`:
https://github.com/Microsoft/generator-docker/pull/49#issuecomment-170709025
The `rc1-update1` has fixes relevant to dnx.